pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -58,6 +57,8 @@
   private static final String CONSOLE_WEBAPP_USE_HTTPS = "controller.query.console.useHttps";
   private static final String EXTERNAL_VIEW_ONLINE_TO_OFFLINE_TIMEOUT = "controller.upload.onlineToOfflineTimeout";
   private static final String CONTROLLER_MODE = "controller.mode";
+  private static final String NUMBER_OF_CONTROLLER_REPLICAS = "controller.number.replicas";
```

Comment:
Is this number of pinot controller hosts? Not sure why we need to configure this in. Would be best if we did not have to configure number of controller replicas. We should be able to roll out additional controllers as needed.

Reply:
Sure, I'll make it non-configurable for now.

Reply:
Done.

Reply:
Updated.